docs/EN/Configuration/OmnipodDASH.md

```diff
@@ -1,91 +1,567 @@
+
+
+|Resume_1||Resume_2|
+
+2. When the Resume delivery command is successfully confirmed a confirmation dialog will display the message **Insulin delivery has been resumed**. Click **OK** to confirm and proceed.
+
+
+|Resume_3|
+
+3. The **DASH** tab will update the **Pod status (1)** field to display **RUNNING,** and the **Resume Delivery** button will now display the **SUSPEND (2)** button.
+
+
+|Resume_4|
+
+## Acknowledging Pod Alerts
+
+*NOTE - The Silence ALERTS button is only available on the **DASH** tab when the pod expiration or low reservoir alert has been triggered.*
+
+The process below will show you how to acknowledge and dismiss pod beeps when the active pod time reaches the warning time limit before the pod expiration of 72 hours (3 days). This warning time limit is defined in the **Hours before shutdown** Dash alerts setting. The maximum life of a pod is 80 hours (3 days 8 hours), however Insulet recommends not exceeding the 72 hours (3 days) limit.
+
+1. When the defined **Hours before shutdown** warning time limit is reached, the pod will issue warning beeps to inform you that it is approaching its expiration time and pod change will be required soon. You can verify this on the **DASH** tab, the **Pod expires: (1)** field will show the exact time the pod will expire (72 hours after activation), and the text will turn **red** after this time has passed. Under the **Active Pod alerts (2)** field the status message **Pod will expire soon** is displayed. This also will trigger displaying the **SILENCE ALERTS (3)** button.
+
+|ACK_alerts_1|
+
+2. Go to the **DASH** tab and press the **SILENCE ALERTS (2)** button . AAPS sends the command to the pod to deactivate the pod expiration warning beeps and updates the **Pod status (1)** field with **ACKNOWLEDGE ALERTS**.
+
+|ACK_alerts_2|
+
+3. Upon **successful deactivation** of the alerts, **2 beeps** will be issued by the active pod and a confirmation dialog will display the message **Activate alerts have been acknowledged**. Click the **OK** button to confirm and dismiss the dialog.
+
+
+|ACK_alerts_3|
+
+4. Go to the **Omnipod (POD)** tab. Under the **Active Pod alerts** field, the warning message is no longer displayed, and the active pod will no longer issue pod expiration warning beeps.
+
+## View Pod History
+
+This section shows you how to review your active pod history and filter by different action categories. The pod history tool allows you to view the actions and results committed to your currently active pod during its three days (72 - 80 hours) life.
+
+This feature is helpful in verifying boluses, TBRs and basal commands that were sent to the pod. The remaining categories are useful for troubleshooting issues and determining the order of events that occurred leading up to a failure.
+
+*NOTE:*
+**Uncertain** commands will appear in the pod history. However, due to their nature, you cannot ensure their accuracy.
```

Comment:
On Dash only the last command can be uncertain.

We are not sending new commands until the last 'uncertain' command becomes 'confirmed' or 'denied'. The way to 'fix' uncertaing commands is to 'refresh pod status'.